src/utils/dataset-helper.js

```diff
@@ -129,28 +163,47 @@ export const renameSampleFromUrl = (samples, sampleToChange, sampleName) => {
   }
   return sampleName
 }
-
-export const eraseAnnotation = (samples) => {
+export const projectHasDataFile = (typeProject) => {
+  if (
+    "text_entity_recognition" === typeProject ||
+    "text_classification" === typeProject
+  )
+    return "none"
+  if (
+    "video_segmentation" === typeProject ||
+    "image_classification" === typeProject ||
+    "image_segmentation" === typeProject ||
+    "audio_transcription" === typeProject ||
+    "composite" === typeProject ||
+    "data_entry" === typeProject
+  )
+    return "file"
+  return ""
+}
+export const eraseAnnotationAllSamples = (samples) => {
   var samplesWithoutAnnotation = []
   for (let i = 0; i < samples.length; i++) {
     let Newsample = samples[i]
     if (!isEmpty(Newsample.annotation)) {
-      Newsample = setIn(Newsample, ["annotation"], null)
+      delete Newsample["annotation"]
```

Comment:
i think this will probably not work with seamless immutable objects

Reply:
I have test it and it work in normal condition